homeassistant/components/hue/v2/hue_event.py

```diff
@@ -44,11 +70,37 @@ def handle_button_event(evt_type: EventType, hue_resource: Button) -> None:
             CONF_ID: slugify(f"{hue_device.metadata.name}: Button"),
             CONF_DEVICE_ID: device.id,  # type: ignore
             CONF_UNIQUE_ID: hue_resource.id,
-            CONF_TYPE: hue_resource.button.last_event.value,
+            CONF_TYPE: cur_event.value,
             CONF_SUBTYPE: hue_resource.metadata.control_id,
         }
         hass.bus.async_fire(ATTR_HUE_EVENT, data)
 
+        # Handle longpress workaround if needed
+        if (
+            hue_device.product_data.model_id in BTN_WORKAROUND_NEEDED
+            and cur_event == ButtonEvent.INITIAL_PRESS
+        ):
+            asyncio.create_task(handle_longpress_workaround(hue_resource))
+
+    async def handle_longpress_workaround(hue_resource: Button):
+        # Fake `held down` and `long press release` events
```

Comment:
We do not support/allow faking events/data in Home Assistant.

Reply:
It is temporary because of an error in the API

Reply:
nvm, this is not the correct place to fix this